docs/src/layouts/MainLayout.vue

```diff
@@ -1,74 +1,80 @@
 <template>
   <q-layout view="hHh lpR fff" class="bg-lp-dark">
-
-    <q-header class="bg-lp-dark">
-      <q-toolbar>
+    <q-header class="bg-lp-dark horizontal-glow-bar">
+      <q-toolbar class="q-pa-md row justify-between">
         <q-btn flat @click="showDrawer = !showDrawer" round dense icon="menu" v-if="$q.screen.xs" color="lp-primary" />
-        <q-toolbar-title :class="$q.screen.xs? 'row justify-center items-center':''" class="add-vertical-bar position-relative">
-          <q-avatar>
-            <q-icon name="img:homepage-icons/quasar-logo-blue.svg" size="lg"/>
-          </q-avatar>
-          QUASAR
-        </q-toolbar-title>
-        <q-space v-if="$q.screen.gt.xs"/>
+        <div :class="$q.screen.xs? 'row justify-center items-center':''" class="add-vertical-bar position-relative">
+          <picture class="quasar-logo cursor-pointer" @click="$router.push({name: 'home'})">
```

Comment:
Honestly I wouldn't use `picture` for this. It's meant to solve resolution switching and art director problems, which is not the case for our icon
Go with `$q.screen.xx` methods and a simpler v-if, a computed property or similar stuff (`<q-img name="img:xxxx">` was fine I guess, or `img` directly) which allow you to keep Quasar standard viewports

Unless there was something which blocked you from doing this, in that case always add a comment explaining what you tried and why it didn't worked, so others won't repeat your mistakes and have the correct context

Also, use `https://cdn.quasar.dev` endpoint. It redirects onto `https://github.com/quasarframework/cdn` repo and uses GH as a free CDN for Quasar static assets

In your case, here you'll need:
- `https://cdn.quasar.dev/logo-v2/svg/logo-dark.svg`
- `https://cdn.quasar.dev/logo-v2/svg/logo-horizontal-dark.svg`